sdk/core/azure_core_amqp/Test-Cleanup.ps1

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# cspell: ignore JOBID
+
+param (
+  [string]$PackageName
+)
+
+. "$PSScriptRoot\..\..\..\eng\common\scripts\common.ps1"
+
+if (-not $PackageName) {
+  Write-Host "Please provide a package name."
+  exit 1
+}
+
+Write-Host Currently running jobs:
```

Comment:
```suggestion
Write-Host "Currently running jobs:"
```

I'd also rather we don't print jobs, or do it behind a `$VerbosePreference` check. You're assuming we're only running jobs you started. I use jobs for a lot of things - which some job types work system-wide - and don't need to see them all.

Reply:
That's fine - this was primarily diagnostic code to help figure out what was going on in the script - it's not necessary.